Lib/fontTools/subset/__init__.py

```diff
@@ -1983,22 +1983,76 @@ def subset_glyphs(self, s):
 	else:
 		assert False, "unknown 'prop' format %s" % prop.Format
 
+def _paint_glyph_names(paint, colr):
+	result = set()
+
+	def callback(paint):
+		if hasattr(paint, "Glyph"):
+			result.add(paint.Glyph)
```

Comment:
There are sufficiently few for which this is true that paint.Format in ... might suffice?

Reply:
done